linode/manager
# refactor: [M3-7061] - De-Redux-ify the Volumes Drawer

## Description 📝
- Simplifys Volumes drawer logic by no longer using Redux and by breaking one "mega" drawer into many smaller ones

## Major Changes ❗️
- Volumes Drawer is no longer mounted at all times 🎉
- Cleaned up some formik forms
- Improved accessibility by using `<form>` more often


## Preview 📷
> [!NOTE]
> No UI changes are expected

https://github.com/linode/manager/assets/115251059/e80812ff-38af-40a3-82e2-17fde35741b2

## How to test 🧪
- Test all volume actions possible from `/volumes`
  - View Configuration
  - Resize
  - Edit
  - Attach
  - Detach
  - Clone
- Test all volume actions possible on `/linodes/{id}/storage`
  - Create
  - Attach
  - Edit
  - Resize
-  Verify all **Volume** related e2e tests pass
- Verify DC Specific pricing still works as expected

Tests:
```
yarn cy:run -s "cypress/e2e/core/volumes/create-volume-spec.ts, cypress/e2e/core/volumes/resize-volume-spec.ts, cypress/e2e/core/volumes/update-volume-spec.ts"
```
```
yarn test VolumeTableRow.test.tsx
```
